bin/zkEnv.cmd

```diff
@@ -16,7 +16,6 @@ REM limitations under the License.
 
 set ZOOCFGDIR=%~dp0%..\conf
 set ZOO_LOG_DIR=%~dp0%..\logs
-set ZOO_LOG4J_PROP=INFO,CONSOLE
```

Comment:
is there equivalent to set log-levels to o/p mapping in logback?

Reply:
yup that's correct. And that can be done using xml file.